### DISCUSSION TITLE
docs - add installation instructions

### DISCUSSION THREAD

**Discussion by szamanr** [dcastil/tailwind-merge#232] [Ideas]
folks might be confused whether to install this as dependency or dev dependency (e.g. see #99). would be helpful to add a section "installation" which mentions the preferred installation method.

**Comment by dcastil** [comment]
Hey @szamanr! 👋

Thanks for the suggestion!

Whether to put tailwind-merge into dependencies or devDependencies depends on your own use case of the package, so I can't give a definitive answer to this unfortunately and therefore also can't add this info to the docs.

If you aren't sure where to put it, I'd say adding it as a dependency makes sense for most users.

If you want to know more about the difference between dependencies and devDependencies, I recommend checking out the StackOverflow answer here: https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencie/22004559#22004559

**Comment by dcastil** [comment]
FYI Better late than never, but just added the installation instructions to the docs in https://github.com/dcastil/tailwind-merge/pull/486. 😁
